docs/examples/bifacial/plot_bifi_model_mc.py

```diff
@@ -0,0 +1,108 @@
+"""
+Bifacial Modeling - modelchain
+==============================
+
+Example of bifacial modeling using the modelchain
+"""
+
+# %%
+# This example shows how to complete a bifacial modeling example using the
+# :py:class:`pvlib.modelchain.ModelChain` with the
+# :py:func:`pvlib.bifacial.pvfactors_timeseries` function to transpose
+# GHI data to both front and rear Plane of Array (POA) irradiance.
+#
+# Unfortunately ``ModelChain`` does not yet support bifacial simulation
+# directly so we have to do the bifacial irradiance simulation ourselves.
+# Once the combined front + rear irradiance is known, we can pass that
+# to ``ModelChain`` and proceed as usual.
+#
+# Future versions of pvlib may make it easier to do bifacial modeling
+# with ``ModelChain``.
+
+import pandas as pd
+from pvlib import pvsystem
+from pvlib import location
+from pvlib import modelchain
+from pvlib.temperature import TEMPERATURE_MODEL_PARAMETERS as PARAMS
+from pvlib import bifacial
+
+# create site location and times characteristics
+lat, lon = 36.084, -79.817
+tz = 'Etc/GMT+5'
+times = pd.date_range('2021-06-21', '2021-6-22', freq='1T', tz=tz)
+
+# create site system characteristics
+axis_tilt = 0
+axis_azimuth = 180
+gcr = 0.35
+max_angle = 60
+pvrow_height = 3
+pvrow_width = 4
+albedo = 0.2
+bifaciality = 0.75
+
+# load temperature parameters and module/inverter specifications
+temp_model_parameters = PARAMS['sapm']['open_rack_glass_glass']
+cec_modules = pvsystem.retrieve_sam('CECMod')
+cec_module = cec_modules['Trina_Solar_TSM_300DEG5C_07_II_']
+cec_inverters = pvsystem.retrieve_sam('cecinverter')
+cec_inverter = cec_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']
+
+# create a location for site, and get solar position and clearsky data
+site_location = location.Location(lat, lon, tz=tz, name='Greensboro, NC')
+solar_position = site_location.get_solarposition(times)
+cs = site_location.get_clearsky(times)
+
+# load solar position and tracker orientation for use in pvsystem object
+sat_mount = pvsystem.SingleAxisTrackerMount(axis_tilt=axis_tilt,
+                                            axis_azimuth=axis_azimuth,
+                                            max_angle=max_angle,
+                                            backtrack=True,
+                                            gcr=gcr)
+
+# created for use in pvfactors timeseries
+orientation = sat_mount.get_orientation(solar_position['apparent_zenith'],
+                                        solar_position['azimuth'])
+
+# get rear and front side irradiance from pvfactors transposition engine
+irrad = bifacial.pvfactors_timeseries(solar_position['azimuth'],
```

Comment:
Is it worth adding some kind of warning suppression here? On execution, ShapelyDeprecationWarning and RuntimeWarning are raised, by shapely (pfactors dependency) and pvfactors, respectively. It clutters up the example a bit.